setup.py

```diff
@@ -96,9 +96,8 @@ def build_extension(self, ext):
     keywords="molecular dynamics",
     ext_modules=ext_modules,
     packages=find_packages(),
-    python_requires=">=3.7",
+    python_requires=">=3.10",
     install_requires=[
-        "importlib-resources",
         "jax",
         "jaxlib>0.3.13",
```

Comment:
```suggestion
        "jaxlib>=0.4.1",
```

One of the reasons for upgrading was to use new features like the jax.Array type, so presumably we will want at least the version that introduced it?

Reply:
Agreed, but would it make sense to add the constraint in the PR that introduces usage of `jax.Array`?